public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,78 @@
+# Setup Google Drive remote
+
+To setup Google Drive remote you should
+[create and configure Google Project](#create-and-configure-google-project) if
+you don't have one yet.
+
+## Create and Configure Google Project
+
+Since Google Drive has tight API usage quotas, creation and configuration of
+your own `Google Project` is required:
+
+1.  Log into the [Google Cloud Platform](https://console.developers.google.com)
+    account.
+2.  Create `New Project` or select available one.
+3.  Click `ENABLE APIS AND SERVICES` and search for `drive` to enable
+    `Google Drive API` from search results.
+4.  Navigate to
+    [All Credentials](https://console.developers.google.com/apis/credentials)
+    page and click `Create Credentials` to select `OAuth client ID`. It might
+    ask you to setup a product name on the consent screen.
+5.  Select `Other` for `Application type` and click `Create` to proceed with
+    default `Name`.
+6.  `client id` and `client secret` should be showed to you. Use them for
+    further DVC's configuration.
+
+```dvc
+$ dvc remote add myremote gdrive://root/my-dvc-root
+$ dvc remote modify myremote gdrive_client_id my_gdrive_client_id
+$ dvc remote modify myremote gdrive_client_secret gdrive_client_secret
+```
+
+⚠️ It is safe to share `client_id` and `client_secret` at least with your
+teammates or inside organization. These credentialds doesn't give access to your
+Google Drive storage. But Google limits global number of queires per each
+`client_id` and uncontrolled usage of your `cliend_id` by outer world can lead
+to reaching the allowed Google API usage limits.
+
+## Google Drive remote URL format
```

Comment:
or may be we can move this section to the top?

Reply:
I believe that creation of Google Project is the first required user action for now (till DVC will not start sharing default client_id/secret for all users). Have put a comment to look below - https://github.com/iterative/dvc.org/pull/926/commits/9b2559a51a2226f206566a508563b7bab4954b82#diff-17178527abd3f7820652ea6a078a0151R27